docs/references/general/iqdat.md

```diff
@@ -1,27 +1,23 @@
 <!--
 (C) copyright 2020 VT SuperDARN, Virginia Polytechnic Institute & State University
 author: Kevin Sterne
+
+Modifications:
+    2022-11-28 Emma Bland (UNIS) Updated file format description
 -->
 
 # IQDat files
 
 IQDat files are a level-zero type data product that reflect the ADC voltages sampled at the radar site.
 
-## Naming Convenctions
-
-Currently the common naming convention for IQDat files is:
-
-> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.iqdat
+## Naming Convention
 
-SuperDARN radars routinely change operating frequency. Some operating modes use multiple frequencies, either at the same time, or by alternating between frequencies. As a result, some files from some radars include all records for all frequencies in the same file and other radars separate out the individual frequencies into separate files, designated by a channel letter in the file name:
+The community standard for naming IQDAT files is:
 
-> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].iqdat
+> YYYYMMDD.HH.mm.ss.<3-letter radar code>.[a-d].iqdat
 
-For example, on 2019-02-01 the King Salmon radar (KSR) was operating simultaneously on two channels as seen from the file names `20190201.0401.00.ksr.a.iqdat` and `20190201.0401.00.ksr.b.iqdat`.
-Each file contains data from a different frequency channel. 
+The field [a-d] is used to separate data sampled at different frequencies into separate files. This field may not be used if all the available data are provided in a single file.
```

Comment:
This may be an oversimplification - the various `[a-d]` channel identifiers do not always indicate that different frequencies were used (ie it could be different beams or even control programs).

Reply:
```suggestion
The field [a-d] is used when the data have been separated into multiple files based on a particular operating parameter (e.g. frequency, beam pattern, control program). This field may not be used if all the available data are provided in a single file.
```